### PR TITLE
APP-4343 Added styles to close-icon

### PR DESCRIPTION
**Pull request by symphony-annaLlorens**
### Description
- Added styles to the Dropdown close icon for the keyboard navigation. 

### Demo
Screenshot 2021-08-19 at 13 35 48

[Jira-ticket](https://perzoinc.atlassian.net/browse/APP-4343)

### CODE REVIEW

**Review by AliciaSymphony — DISMISSED** [review]

File: src/uitoolkit-components/_select.scss [hunk 1]
```diff
@@ -158,6 +158,16 @@
         &:hover {
           color: $--tk-input-default-active-color;
         }
+        .tk-select__close-icon {
+          border-width: toRem(2);
+          border-radius: 50%;
+          display: inline-flex;
+          padding: toRem(1);
+          &:focus-visible {
+            border: toRem(2) solid $focus-ring-browser-default;
```
Comment (symphony-jean-michael) [inline src/uitoolkit-components/_select.scss:167]: Did you try to use the mixin 'browserOutline'? (FYI, the checkbox uses it.)
Mixin: https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-styles/blob/fa695d4d67d68cae9b2d834741eaa278815188d7/src/utils/mixins/_common.scss#L17
Example: 
https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-styles/blob/fa695d4d67d68cae9b2d834741eaa278815188d7/src/utils/mixins/_checkbox_radio_common.scss#L12

Comment (AliciaSymphony) [reply src/uitoolkit-components/_select.scss:167]: Unfortunately -webkit-focus-ring-color cannot be used on border property

Comment (AliciaSymphony) [reply src/uitoolkit-components/_select.scss:167]: And the `outline` property has a square shape... so to follow the design it is not possible to use `outline`

**Review by symphony-jean-michael — APPROVED** [review]

**Comment by AliciaSymphony** [comment]
image
oups

File: src/variables/colors/_statics.scss [hunk 2]
```diff
@@ -100,6 +100,11 @@ $colors: (
   @return map-get($color, $num);
 }
 
+/** Focus ring color browser
+/** This should be removed with:
+/** - APP-4442: Double check focus styles between Elements and C2OPEN  and APP-4441: FE: [NEXUS] Align styles focus on every elementsOPEN */
+$focus-ring-browser-default: #005fcc;
```
Comment (symphony-jean-michael) [inline src/variables/colors/_statics.scss:106]: @symphony-annaLlorens  This variable is not used anymore ;)

**Review by AliciaSymphony — APPROVED** [review]
LGTM a bit of a shame that outline doesn't propose a radius property

**Pull request merged**